File: python_leaoai/models/Pessoa/Usuario.py
# models/usuario.py

from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
from sqlalchemy.orm import Mapped, mapped_column
from typing import Optional
from datetime import datetime
from database import Base  # Assumindo que você tenha um Base comum

class Usuario(Base):
    __tablename__ = "usuario"

    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
    nome: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    login: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    email: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    password: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    role_gate: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    avatar: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    imagem: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    password_token: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    password_token_expiry: Mapped[Optional[datetime]] = mapped_column(DateTime, nullable=True)
    subscription_id: Mapped[Optional[int]] = mapped_column(Integer, nullable=True)
    journey_id: Mapped[Optional[int]] = mapped_column(Integer, nullable=True)
    guid: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    token: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    data_nascimento: Mapped[Optional[datetime]] = mapped_column(DateTime, nullable=True)
    genero: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    logradouro: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    cep: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    cidade: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    estado: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    bairro: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    complemento: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    numero: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    phone: Mapped[Optional[str]] = mapped_column(String, nullable=True)
    cpf: Mapped[Optional[str]] = mapped_column(String, nullable=True)

File: Framework.Data/Models/Pessoa/Usuario.cs
using System;

namespace Data.Models.Pessoa
{
	public class Usuario : Base
    {
        public string Nome { get; set; }
        public string Login { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string RoleGate { get; set; }
        public string Avatar { get; set; }
        public string Imagem { get; set; }
        public string PasswordToken { get; set; } 
        public DateTime? PasswordTokenExpiry { get; set; }
        public int? SubscriptionId { get; set; } 
        public int? JourneyId { get; set; }
        public string Guid { get; set; }
        public string Token { get; set; }

        public DateTime DataNascimento { get; set; }

        public string Genero { get; set; }
        public string Logradouro { get; set; }

        public string CEP { get; set; }

        public string Cidade { get; set; }

        public string Estado { get; set; }

        public string Bairro { get; set; }

        public string Complemento { get; set; }

        public string Numero { get; set; }

        public string Phone { get; set; }

        public string Cpf { get; set; }


   
    }
}
